app/jobs/zendesk_tickets_job.rb

```diff
@@ -9,12 +9,16 @@ def perform(feedback)
         email: feedback.email,
         name: 'Unknown'
       },
-      custom_fields: [
-        { id: '23730083', value: feedback.referrer },
-        { id: '23757677', value: 'prison_visits' },
-        { id: '23791776', value: feedback.user_agent },
-        { id: '23984153', value: feedback.prison }
-      ]
+      custom_fields: custom_fields(feedback)
     )
   end
+
+  def custom_fields(feedback)
+    [
+      { id: '23730083', value: feedback.referrer },
+      { id: '23757677', value: 'prison_visits' },
+      { id: '23791776', value: feedback.user_agent },
+      { id: '23984153', value: feedback.prison }
+    ]
+  end
```

Comment:
I'd prefer if we weren't hardcoding zendesk metadata in an instance  method at all.  Perhaps put it in Rails.config or an initialiser?